components/brave_wallet/renderer/js_cardano_wallet_api.cc

```diff
@@ -33,7 +26,13 @@ v8::Local<v8::Value> ConvertError(
     v8::Isolate* isolate,
     const v8::Local<v8::Context>& context,
     const mojom::CardanoProviderErrorBundlePtr& error) {
-  CHECK(error);
```

Comment:
i wonder what has happened here? so it was completely illegal before and now just ok?

Reply:
mojom function now return two optionals like this`(array<string>? used_addresses, CardanoProviderErrorBundle? error)`

so we can safely handle null error in this way:
```
if (used_addresses) {
  foo(*used_addresses...);
} else {
   bar(ConvertError(error));
}
```